rlclientlib/live_model_impl.cc

```diff
@@ -54,6 +58,10 @@ int live_model_impl::init(api_status* status)
 {
   RETURN_IF_FAIL(init_trace(status));
   RETURN_IF_FAIL(init_model(status));
+
```

Comment:
This seems like an abstraction mismatch.  Perhaps local_loop initialization needs to be refactored from remote_loop init.  local_loop init can then impose conditions on acceptable configurations.